src/stabilization_report_template.md

```diff
@@ -0,0 +1,105 @@
+>
+> You can copy the following template after the separator and edit it as Markdown, replacing the *TODO* placeholders with answers. 
+
+---
+
+> ## General design
+
+>  ### What is the RFC for this feature and what changes have occurred to the user-facing design since the RFC was finalized?
+
+*TODO*
+
+>  ### What behavior are we committing to that has been controversial? Summarize the major arguments pro/con.
+
+*TODO*
+
+> ### Are there extensions to this feature that remain unstable? How do we know that we are not accidentally committing to those?
+
+*TODO*
+
+> ## Has a Call for Testing period been conducted? If so, what feedback was received?
+>
+> Does any OSS nightly users use this feature? For instance, a useful indication might be "search <grep.app> for `#![feature(FEATURE_NAME)]` and had `N` results".
+
+*TODO*
+
+> ## Implementation quality
+
+*TODO*
+
+> ### Summarize the major parts of the implementation and provide links into the code (or to PRs)
+>
+> An example for async closures: <https://rustc-dev-guide.rust-lang.org/coroutine-closures.html>.
+
+*TODO*
+
+> ### Summarize existing test coverage of this feature
+>
+> Consider what the "edges" of this feature are.  We're particularly interested in seeing tests that assure us about exactly what nearby things we're not stabilizing.
+>
+> Within each test, include a comment at the top describing the purpose of the test and what set of invariants it intends to demonstrate. This is a great help to those reviewing the tests at stabilization time.
+>
+> - What does the test coverage landscape for this feature look like?
+>   - Tests for compiler errors when you use the feature wrongly or make mistakes?
+>   - Tests for the feature itself:
+>       - Limits of the feature (so failing compilation)
+>       - Exercises of edge cases of the feature
+>       - Tests that checks the feature works as expected (where applicable, `//@ run-pass`).
+>   - Are there any intentional gaps in test coverage?
+>
+> Link to test folders or individual tests (ui/codegen/assembly/run-make tests, etc.).
+
+*TODO*
+
+> ### What outstanding bugs in the issue tracker involve this feature? Are they stabilization-blocking?
+
+*TODO*
+
+> ### What FIXMEs are still in the code for that feature and why is it ok to leave them there?
+
+*TODO*
+
+> ### Summarize contributors to the feature by name for recognition and assuredness that people involved in the feature agree with stabilization 
+
+*TODO*
+
+> ### Which tools need to be adjusted to support this feature. Has this work been done?
+>  
+> Consider rustdoc, clippy, rust-analyzer, rustfmt, rustup, docs.rs.
```

Comment:
```suggestion
> Consider rustdoc (both JSON and HTML), clippy, rust-analyzer, rustfmt, rustup, docs.rs.
```

Even though rustdoc JSON is unstable, the new feature must be visible in the JSON output or else cargo-semver-checks cannot lint it.

In an ideal world, stabilized features would ship with a full set of SemVer lints concurrently with the stable Rust release that first includes the feature. Getting the data into rustdoc JSON is a prerequisite for that ideal future.

Reply:
Thanks, included this.